pytube/contrib/channel.py

```diff
@@ -199,3 +199,99 @@ def _extract_videos(raw_json: str) -> Tuple[List[str], Optional[str]]:
             ),
             continuation,
         )
+
+    @staticmethod
+    def _extract_playlists(raw_json: str) -> Tuple[List[str], Optional[str]]:
+        """Extracts playlists from a raw json page
+
+        :param str raw_json: Input json extracted from the page or the last
+            server response
+        :rtype: Tuple[List[str], Optional[str]]
+        :returns: Tuple containing a list of up to 100 video watch ids and
+            a continuation token, if more videos are available
+        """
+        initial_data = json.loads(raw_json)
+
+        # this is the json tree structure, if the json was extracted from
+        # html
+        try:
+            playlists = initial_data["contents"][
+                "twoColumnBrowseResultsRenderer"][
+                "tabs"][2]["tabRenderer"]["content"][
+                "sectionListRenderer"]["contents"][0][
+                "itemSectionRenderer"]["contents"][0][
+                'shelfRenderer']["content"][
+                'horizontalListRenderer']["items"]
+        except (KeyError, IndexError, TypeError):
+            playlists = []
```

Comment:
probably an info warning to explain why it's empty.

Reply:
No need for an info warning -- this is going to happen on e.g. channels that don't have any playlists created on them. It makes more sense to return an empty array than an error.

Reply:
@tfdahlin  hmm; consideration would be as a user running this on like 40 channels and having some of those be empty --> what would feedback mechanism be to help user debug?

Reply:
Yeah, we should probably remove the try-catch block honestly